resources/eventing/charts/controller/values.yaml

```diff
@@ -0,0 +1,18 @@
+# override name to avoid collision with kantive eventing resources
+nameOverride: subscription-controller
+replicaCount: 1
+serviceAccount:
+  # name defines optionally another name than the default name for the service account
+  name: ""
+
+clusterRole:
+  # name defines optionally another name than the default name for the cluster role
+  name: ""
+
+image:
+  # name is the name of the container image for the event-publisher-proxy
```

Comment:
The event-publisher-proxy is mentioned in the comments. I guess you want to update it to match the controller instead.